CPP11.md

```diff
@@ -869,6 +870,18 @@ foo(std::make_shared<T>(), function_that_throws(), std::make_shared<T>());
 
 See the section on [smart pointers](#smart-pointers) for more information on `std::unique_ptr` and `std::shared_ptr`.
 
+### std::ref
+`std::ref(val)` is used to create object of type `std::reference_wrapper` that holds reference of val. Used in cases when usual reference passing using `&` does not compile or `&` is dropped due to type deduction.
+
+```c++
+// create a container to store reference of objects.
+auto val = 99;
+vector<reference_wrapper<int> >vec; // vector<int&>vec does not compile
```

Comment:
A couple things:
1. use `std::vector`, and `std::reference_wrapper`
2. Remove the space from the right angle bracket, it's no longer required.

```
std::vector<std::reference_wrapper<int>> vec;
```